server/api/apiSpecification.yaml

```diff
@@ -0,0 +1,568 @@
+openapi: 3.0.0
+info:
+  description: >
+    _This document describes the REST API of Ark Automate._
+    Ark Automate is a platform that allows office users and software developers to automate business or everyday processes by simply sketching the steps of their process. By using simple flowcharts or powerful BPMN in their process outlines, users can create small software solutions using RPA that finish their tasks much faster and more reliably.
+  version: 1.0.0
+  title: Ark API
+  license:
+    name: LICENSE (MIT)
+    url: https://github.com/bptlab/ark_automate
+
+servers:
+  - url: '{protocol}://{server}/api/v1'
+    variables:
+      protocol:
+        default: https
+        enum: [http, https]
+      server:
+        default: localhost:5000
+
+tags:
+  - name: RPA-Functionalities
+    description: Operations about rpa supported applications, tasks and parameters
+  - name: Robots
+    description: Operations about robots
+
+paths:
+  #### RPA Functionalities ####
+  /functionalities:
+    #GET /rpa-framework/commands/get-available-applications
+    get:
+      tags:
+        - RPA-Functionalities
+      summary: Get all functionality objects including task, application and parameters
+      operationId: getFunctionalities
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Functionalities'
+  /functionalities/applications:
+    #// GET /rpa-framework/commands/get-available-tasks-for-application?application=Browser
+    get:
+      tags:
+        - RPA-Functionalities
+      summary: Get all applications that ark automate supports
+      operationId: getApplications
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Applications'
+  /functionalities/{application}/tasks:
+    parameters:
+      - name: application
+        in: path
+        description: The name of an application
+        required: true
+        schema:
+          $ref: '#/components/schemas/Applications'
+    # GET /ssot/:id
+    get:
+      tags:
+        - RPA-Functionalities
+      summary: Gets all the tasks to execute for an application
```

Comment:
Is this correct? I think this description could be better. Especially if its for `# GET /ssot/:id` I don't think it describes the right functioality.

Reply:
The description is right. I just mixed up the comment here. It should be `#// GET /rpa-framework/commands/get-available-tasks-for-application?application=Browser` and not `# GET /ssot/:id`